src/ducks/account/AccountSwitch.styl

```diff
@@ -7,16 +7,10 @@
 
 .account-switch
     position relative
-    padding-bottom 20px
-
-    a
-    a:visited
-        color            dodgerBlue
-        text-decoration  none
-
-    a:hover
-    a:active
-        text-decoration  underline
+    top 0
+    padding 0
+    border 0
+    margin 0
```

Comment:
What are the default values ? Are they really necessary ?

Reply:
This is a cut and paste but I had forgotten the deleted part